modules/nf-core/bcftools/pluginimputeinfo/meta.yml

```diff
@@ -0,0 +1,68 @@
+---
+name: "bcftools_pluginimputeinfo"
+description: Adds imputation information metrics to the INFO field based on selected FORMAT tags. Only the IMPUTE2 INFO metric from FORMAT/GP tags is currently available.
+keywords:
+  - impute-info
+  - bcftools
+  - VCF
+tools:
+  - "bcftools":
+      description: "BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations."
+      homepage: "http://samtools.github.io/bcftools/bcftools.html"
+      documentation: "https://samtools.github.io/bcftools/howtos/plugins.html"
+      tool_dev_url: "https://github.com/samtools/bcftools"
+      doi: "10.1093/bioinformatics/btp352"
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: |
+        The vcf file to be inspected.
+        e.g. 'file.vcf'
+  - index:
+      type: file
+      description: |
+        The tab index for the VCF file to be inspected.
+        e.g. 'file.tbi'
+  - regions:
+      type: file
+      description: |
+        Optionally, restrict the operation to regions listed in this file.
+        e.g. 'file.vcf'
+  - targets:
+      type: file
+      description: |
+        Optionally, restrict the operation to regions listed in this file (doesn't rely upon index files)
+        e.g. 'file.vcf'
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
+      description: VCF normalized output file
```

Comment:
A bit more information here would be nice, normalised how? I thought the module is adding extra meta info